cypress/e2e/about.cy.ts

```diff
@@ -51,9 +59,17 @@ describe('About page', () => {
     })
 
     context('Portrait mode', () => {
-        beforeEach(() => {
-            cy.viewport('iphone-5')
+        before(() => {
             cy.visit('/about')
+            // This wait time is to give the page time to load from Prod when ran in CI.
+            cy.wait(3000)
```

Comment:
do we really need 3 secs for every test group? it's gonna be so slow

Reply:
@ermish It can take that long for the page to fully render when it's ran in the prod environment (like it's done by our GitHub actions). That's why it sometimes fails and sometimes doesn't. Maybe you might know how to do this, but we could shorten the wait time if the tests are running against the local dev server instead of prod. That would at least make them run more quickly while developing. I'm not sure how to check for that in Cypress and set the conditional.

Reply:
let's chat in a future issue about how to optimize it